tracer/test/test-applications/azure-functions/Samples.AzureFunctions.AllTriggers/AllTriggers.cs

```diff
@@ -39,6 +39,7 @@ public void TimerTrigger([TimerTrigger(EveryTenSeconds)] TimerInfo myTimer, ILog
             [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = "simple")] HttpRequest req,
             ILogger log)
         {
+            await Task.CompletedTask; // Unnecessary, but allows compiler to not raise CS1998: This async method lacks 'await'
```

Comment:
```suggestion
            await Task.Yield();
```

This way it will have a purpose **and** it won't trigger my OCDs 😅